.github/workflows/main.yml

```diff
@@ -296,11 +296,12 @@ jobs:
           ./scripts/generatePiImage.sh
 
       # Upload final fat jar as artifact.
-      - uses: actions/upload-artifact@master
+      - uses: actions/upload-artifact@v3
         with:
           name: jar
           path: photon-server/build/libs
-      - uses: actions/upload-artifact@master
+      - uses: actions/upload-artifact@v3
+        if: github.event_name != 'pull_request'
         with:
```

Comment:
Why not on PRs anymore?

Reply:
The image is not created on PRs, and it was causing a warning.